<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqClientMainWindow</class>
 <widget class="QMainWindow" name="pqClientMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>762</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Qt/Components/Resources/pqComponents.qrc">
    <normaloff>:/pqWidgets/Icons/pvIcon512.png</normaloff>:/pqWidgets/Icons/pvIcon512.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="pqTabbedMultiViewWidget" name="MultiViewWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1152</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSources">
    <property name="title">
     <string>&amp;Sources</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFilters">
    <property name="title">
     <string>Fi&amp;lters</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Macros">
    <property name="title">
     <string>&amp;Macros</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Catalyst">
    <property name="title">
     <string>&amp;Catalyst</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menuSources"/>
   <addaction name="menuFilters"/>
   <addaction name="menuTools"/>
   <addaction name="menu_Catalyst"/>
   <addaction name="menu_Macros"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="pqStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="pipelineBrowserDock">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Pipeline Browser</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="pqPipelineBrowserWidget" name="pipelineBrowser">
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="statisticsDock">
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Statistics Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="pqDataInformationWidget" name="statisticsView"/>
  </widget>
  <widget class="QDockWidget" name="animationViewDock">
   <property name="windowTitle">
    <string>Animation View</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="pqPVAnimationWidget" name="animationView"/>
  </widget>
  <widget class="QDockWidget" name="comparativePanelDock">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Comparative View Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="pqComparativeVisPanel" name="comparativeVisPanel"/>
  </widget>
  <widget class="QDockWidget" name="collaborationPanelDock">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Collaboration Panel</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="pqCollaborationPanel" name="collaborationPanel"/>
  </widget>
  <widget class="QDockWidget" name="informationDock">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Information</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="informationWidgetFrame">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QScrollArea" name="informationScrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="pqProxyInformationWidget" name="informationWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>78</width>
          <height>154</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="memoryInspectorDock">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Memory Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="pqMemoryInspectorPanel" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="propertiesDock">
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>41</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="pqPropertiesPanel" name="propertiesPanel"/>
  </widget>
  <widget class="QDockWidget" name="multiBlockInspectorDock">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Multi-block Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="pqMultiBlockInspectorWidget" name="multiBlockInspectorPanel"/>
  </widget>
  <widget class="QDockWidget" name="lightInspectorDock">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Light Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="pqLightsInspector" name="lightInspectorPanel"/>
  </widget>
  <widget class="QDockWidget" name="catalystInspectorDock">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Catalyst Export Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="colorMapEditorDock">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Color Map Editor</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="pqColorMapEditor" name="colorMapEditorPanel"/>
  </widget>
  <widget class="QDockWidget" name="selectionDisplayDock">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Selection Display Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="pqFindDataSelectionDisplayFrame" name="selectionDisplayWidget">
    <property name="useVerticalLayout" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="displayPropertiesDock">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Display</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="pqPropertiesPanel" name="displayPropertiesPanel">
    <property name="panelMode" stdset="0">
     <number>2</number>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="viewPropertiesDock">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>View</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="pqPropertiesPanel" name="viewPropertiesPanel">
    <property name="panelMode" stdset="0">
     <number>4</number>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="timeInspectorDock">
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Time Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="pqTimeInspectorWidget" name="timeInspectorPanel"/>
  </widget>
  <widget class="QDockWidget" name="outputWidgetDock">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Output Messages</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="pqOutputWidget" name="outputWidget">
    <property name="settingsKey" stdset="0">
     <string>OutputMessages</string>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="pythonShellDock">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Python Shell</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="pythonShellDummy">
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Python support not available!</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqPipelineBrowserWidget</class>
   <extends>QWidget</extends>
   <header>pqPipelineBrowserWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqTabbedMultiViewWidget</class>
   <extends>QWidget</extends>
   <header>pqTabbedMultiViewWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqPVAnimationWidget</class>
   <extends>QWidget</extends>
   <header>pqPVAnimationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqDataInformationWidget</class>
   <extends>QWidget</extends>
   <header>pqDataInformationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqStatusBar</class>
   <extends>QStatusBar</extends>
   <header>pqStatusBar.h</header>
  </customwidget>
  <customwidget>
   <class>pqComparativeVisPanel</class>
   <extends>QWidget</extends>
   <header>pqComparativeVisPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqCollaborationPanel</class>
   <extends>QWidget</extends>
   <header>pqCollaborationPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqProxyInformationWidget</class>
   <extends>QWidget</extends>
   <header>pqProxyInformationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqMemoryInspectorPanel</class>
   <extends>QWidget</extends>
   <header>pqMemoryInspectorPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqPropertiesPanel</class>
   <extends>QWidget</extends>
   <header>pqPropertiesPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqColorMapEditor</class>
   <extends>QWidget</extends>
   <header>pqColorMapEditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqFindDataSelectionDisplayFrame</class>
   <extends>QWidget</extends>
   <header>pqFindDataSelectionDisplayFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqTimeInspectorWidget</class>
   <extends>QWidget</extends>
   <header>pqTimeInspectorWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqOutputWidget</class>
   <extends>QWidget</extends>
   <header>pqOutputWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqMultiBlockInspectorWidget</class>
   <extends>QWidget</extends>
   <header>pqMultiBlockInspectorWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqLightsInspector</class>
   <extends>QWidget</extends>
   <header>pqLightsInspector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqCatalystExportInspector</class>
   <extends>QWidget</extends>
   <header>pqCatalystExportInspector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Qt/Components/Resources/pqComponents.qrc"/>
 </resources>
 <connections/>
</ui>
